### PR TITLE
Refactor Postman Docs

### PR DESCRIPTION
**Pull request by laviniat1996**
This PR adds more info on how to use Postman, such as:
- how to make API calls with video examples
- info about Postman variables
- how to use data visualization with video examples

### CODE REVIEW

**Comment by meaghanfitzgerald** [comment]
Hey @laviniat1996 the build is failing every check, please run `yarn build` on your local branch to see why and push the changes when the build passes. Instructions were in the [pull_request_template](https://github.com/ava-labs/avalanche-docs/blob/master/.github/pull_request_template.md)

**Review by meaghanfitzgerald — DISMISSED** [review]
Hey @laviniat1996 the build is failing every check, please run yarn build on your local branch to see why and push the changes when the build passes. Instructions were in the [pull_request_template](https://github.com/ava-labs/avalanche-docs/blob/master/.github/pull_request_template.md)

File: docs/tooling/avalanchego-postman-collection/variables.md [hunk 1]
```diff
@@ -0,0 +1,168 @@
...
+
+![Variables Scope](/img/postman/postman-19-set-collection-var.png)
+
+Now it can be used on any call from this collection
+
+
+
+
+
+### Set an Environment Variable
```
Comment (meaghanfitzgerald) [inline docs/tooling/avalanchego-postman-collection/variables.md:121]:
```suggestion
### Set an Environment Variable
```

Comment (meaghanfitzgerald) [inline docs/tooling/avalanchego-postman-collection/variables.md:121]:
```suggestion
### Set Environment Variables
```

**Comment by meaghanfitzgerald** [comment]
any update on this?

**Comment by laviniat1996** [comment]
> any update on this?
Sorry, I didn't have time to work on this last week. I feel like the issue could be  Cloudflare Pages not allowing hosting for big video files. They have Cloudflare Stream too, so I suppose that's the option for adding videos. Not sure tho.

**Comment by meaghanfitzgerald** [comment]
Definitely wont merge if cloudflare wont build, is it possible to remove the videos and instead embed a youtube video like this: https://docs.avax.network/learn/avalanche/avalanche-consensus#deep-dive-into-avalanche-consensus

**Comment by laviniat1996** [comment]
@meaghanfitzgerald this is finally ready

**Review by dubrado — APPROVED** [review]

**Review by meaghanfitzgerald — APPROVED** [review]
lgtm, thanks!

**Pull request merged**